### PR TITLE
Edge CSS Custom Properties Support

### PR DESCRIPTION
**Pull request by arjunyel**
Edge now supports [CSS Custom Properties](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/csscustompropertiesakacssvariables/)

### CODE REVIEW

**Comment by arjunyel** [comment]
@kfranqueiro good catch, I updated the pull request

File: docs/getting-started.md [hunk 1]
```diff
@@ -235,7 +235,7 @@ following to the `<head>` tag of `index.html`:
 </style>
 ```
 
-If you are using any modern browser besides Edge (which is currently [working on it](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/csscustompropertiesakacssvariables/)), you'll see that the button background as well as the focused underline and label on text
+If you're using any supported browser, besides IE 11, you'll see that the button background as well as the focused underline and label on text
```
Comment (kfranqueiro) [inline docs/getting-started.md]: Nit: remove extra comma after "browser"; IMO the sentence flows better with just the one comma after "IE 11".

Comment (Garbee) [inline docs/getting-started.md]: The extra comma here is perfectly correct grammatically. "besides IE11" in this case is an appositive, which are denoted with *surrounding* commas.

Comment (kfranqueiro) [inline docs/getting-started.md]: Ah, point taken. Disregard that nit then :)

**Comment by kfranqueiro** [comment]
I'm going to go ahead and merge this documentation update. However, we're aware of some issues related to Edge, some of which may be due to issues with its current implementation of CSS custom properties (see known issues listed on [caniuse](http://caniuse.com/#search=variables)). We are tracking these in Pivotal, but don't have a timeline on fixing them.

**Pull request merged**